src/containers/shared/components/Transaction/EscrowCancel/TableDetail.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+export const TableDetail = (props: any) => {
+  const { t } = useTranslation()
+  const { instructions } = props
+  const { owner, sequence, ticketSequence } = instructions
+  return (
+    <div className="escrow">
+      {owner && (
+        <div>
+          <span className="label">{t('cancel_escrow')}</span>
+          <span className="account">{owner}</span>
+          <span>
+            -{sequence !== 0 ? sequence : `${ticketSequence} (Ticket)`}
```

Comment:
nit: In EscrowCancel/Simple.tsx there was a space after the `-` character, but there isn't one here. Not sure if there's a standard spacing rule we're aiming to follow with dashes.